test/dynamo/test_modules.py

```diff
@@ -703,6 +704,9 @@ def test_generation_tag(self):
         m3 = deepcopy(m1)
         self.assertEqual(GenerationTracker.get_generation_value(m3), cur_generation)
 
+    # torch._subclasses.fake_tensor.UnsupportedFakeTensorException: meta converter nyi
+    # due to custom subclass (TensorProxy)
+    @unittest.expectedFailure
```

Comment:
I have a fix for this in a local patchset, this xfail AOK

Reply:
k